### PR TITLE
Duplicate Blank Page Issue

### LINKED ISSUE #997: Duplicate default pages issue
How to reproduce:

1. go through the wizard with 1 blank page selects
2. at summary page, change the framework
3. then go to the home page and click quick start button, which leads you to the summary page
4. then go to the pages page, you would say two blank pages get defaulted 

image

### PR DESCRIPTION
**Pull request by streetcoder123**
Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/32097

**How to reproduce the Bug**
- Outlined in the issue: #997 

**Changes**
- CoreTemplateStudio CLI was is not able to handle multiple task requests at the same time.
- So, a queue system is implemented (using promise chaining) to allow CoreTS CLI to run all the tasks in a sequential manner.

**Result**
- Follow the steps in the reported issue. The bug should no longer be there.

closes #997

### CODE REVIEW

File: src/extension/src/coreTemplateStudio.ts [hunk 1]
```diff
@@ -94,57 +99,77 @@ export class CoreTemplateStudio {
...
 
   public async sync(payload: ICommandPayload): Promise<any> {
-    const syncCommand = `${CONSTANTS.CLI.SYNC_COMMAND_PREFIX} -p ${payload.payload.path}\n`;
-    this._processCli.stdin.write(syncCommand);
-    this.cliEvents.on(CONSTANTS.CLI.SYNC_PROGRESS_STATE, (data) => {
-      payload.liveMessageHandler(data['status'], data['progress']);
+    const syncCommand = `${CONSTANTS.CLI.SYNC_COMMAND_PREFIX} -p ${
+      payload.payload.path
+    }\n`;
+    this.cliEvents.on(CONSTANTS.CLI.SYNC_PROGRESS_STATE, data => {
```
Comment (jcoc611-microsoft) [inline src/extension/src/coreTemplateStudio.ts:145]: This part also has to be in the promise chain

Comment (streetcoder123) [reply src/extension/src/coreTemplateStudio.ts:145]: As we talked, we can leave this as is for now :)

**Review by Tanya0609 — DISMISSED** [review]
great work!

File: src/extension/src/coreTemplateStudio.ts [hunk 2]
```diff
@@ -59,7 +60,12 @@ export class CoreTemplateStudio {
       cliExecutableName
     );
 
-    let cliWorkingDirectory = path.join(extensionPath, "src", "corets-cli", platform);
+    let cliWorkingDirectory = path.join(
```
Comment (SahilTara) [inline src/extension/src/coreTemplateStudio.ts]: Small thing make this const. Other than that I think the code is good!

**Review by jcoc611-microsoft — APPROVED** [review]
Great! Simple queues

**Review by Tanya0609 — APPROVED** [review]

**Review by jsondoo — APPROVED** [review]
good work :)

**Pull request merged**